ming_sim/cli/terminal.py

```
@@ -551,70 +530,64 @@ def _trail_extraction_after_reply_cli(
-            print("待补账本已补写完毕。\n")
-    except Exception as exc:
-        print(f"重试补写失败：{exc}\n")
+        # #1353 fold-in r5：欠账唯一处理路=过月内部 drain；禁玩家可见技术提示。
+        pass
+    finally:
+        if ticket is not None:
+            try:
+                from ming_sim.session_write_queue import get_session_write_queue
+                get_session_write_queue(session).complete(ticket)
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)